NRLMMD-GEOIPS/geoips
# 352 auto build and deploy sbin and wheels for releases

# Reviewer Checklist

PR author: Please ensure you meet all of the below requirements, and check boxes appropriately.

Reviewers: Please confirm all required testing/documentation has been completed prior to approving.

* [x] Required ***unit tests*** added and pass for new/modified functionality
  * [x] None required (explain if checked, delete line if not checked)
  * [x] None required. (explain if checked, delete line if not checked)
  * [x] None required (explain if checked, delete line if not checked)

There is no need for new tests for this functionality. It is "tested" automatically by the CI process. I have, however, ensured that, when GeoIPS is installed via `pip install geoips==1.11.3b1` it is still able to run `base_test.sh`.

https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#352


# Testing Instructions

I think that this is well-tested, but if you wanted to test it, you could create a new release from this branch, then check to be sure that the new version is made available on pypi.org.

# Summary

GitHub Actions Updates
======================

Add a workflow that builds sdist and wheel packages for new releases and pushes them to
Pypi.

::

    .github/workflows/package-and-publish.yaml

# Output
